### DISCUSSION TITLE
Changedetection lxc - Browsersteps "object is not subscriptable"

### DISCUSSION THREAD

**Discussion by psilantropy** [tteck/Proxmox#3549] [Help Needed]
![image](https://github.com/user-attachments/assets/d80ff9fa-d922-41be-a61a-950ed3051f00)

Anyone have this issue with the changedetection lxc? (likely not a script problem)
I've tried this a few months ago without issue. Deployed a few times on different nodes to test.

The problem seems related to the network/hostname in the docker network - https://github.com/dgtlmoon/changedetection.io/issues/1823

I thought maybe the ws://browserless-chrome:3000 was unreachable, not sure. + hostname didn't resolve it.

**Comment by psilantropy** [comment]
I'm almost certain this is a DNS issue. I'll try resolve and come back to close if so.
Edit: No, not dns.

**Reply by olsonnn** [reply]
> I'm almost certain this is a DNS issue. I'll try resolve and come back to close if so. Edit: No, not dns.

same issue here. Latest proxmox. Did you found a solution?

edit found it!
mv /opt/browserless /opt/browserless_old
git clone -b v1 https://github.com/browserless/chrome /opt/browserless
npm install --prefix /opt/browserless
npm run build --prefix /opt/browserless
npm prune production --prefix /opt/browserless

and add it to /etc/hosts
127.0.0.1  browserless-chrome

then
systemctl start browserless.service

and it all works

**Reply by psilantropy** [reply]
Nice one, thank you. I haven't touched this container since posting.
I'll try that soon :)
